zbarcam/zbarcam.c

```diff
@@ -51,20 +51,21 @@ static const char *note_usage = N_(
     "\n"
     "scan and decode bar codes from a video stream\n"
     "\n"
-    "options:\n"
-    "    -h, --help      display this help text\n"
-    "    --version       display version information and exit\n"
-    "    -q, --quiet     disable beep when symbol is decoded\n"
-    "    -v, --verbose   increase debug output level\n"
-    "    --verbose=N     set specific debug output level\n"
-    "    --xml           use XML output format\n"
-    "    --raw           output decoded symbol data without converting charsets\n"
-    "    -1, --oneshot   exit after scanning one bar code\n"
-    "    --nodisplay     disable video display window\n"
+    "options:\n" 
+    "    -h, --help       display this help text\n"
+    "    --version        display version information and exit\n"
+    "    -q, --quiet      disable beep when symbol is decoded\n"
+    "    -v, --verbose    increase debug output level\n"
+    "    --verbose=N      set specific debug output level\n"
+    "    --xml            use XML output format\n"
+    "    --raw            output decoded symbol data without converting charsets\n"
+    "    --rawnodelimiter same as --raw, but without newlines between symbol data\n"
+    "    -1, --oneshot    exit after scanning one bar code\n"
+    "    --nodisplay      disable video display window\n"
     "    --prescale=<W>x<H>\n"
-    "                    request alternate video image size from driver\n"
+    "                     request alternate video image size from driver\n"
     "    -S<CONFIG>[=<VALUE>], --set <CONFIG>[=<VALUE>]\n"
-    "                    set decoder/scanner <CONFIG> to <VALUE> (or 1)\n"
+    "                     set decoder/scanner <CONFIG> to <VALUE> (or 1)\n"
     /* FIXME overlay level */
```

Comment:
Why are you adding an extra space on lines 66/68?

Reply:
Because --rawnodelimiter is really long (the new longest option), and things needed shifted over to align.

Also, IIRC you're using a mixture of tabs and spaces, which was a little weird.